core/src/test/java/io/questdb/test/griffin/engine/window/WindowFunctionTest.java

```diff
@@ -218,17 +222,18 @@ public void testFrameFunctionOverNonPartitionedRangeWithLargeFrame() throws Exce
             execute("insert into tab select (100000+x)::timestamp, x/4, x, 'k' || (x%5) ::symbol, x*2::double, 'k' || x from long_sequence(90000)");
 
             assertQueryNoLeakCheck(
-                    "ts\ti\tj\tavg\tsum\tfirst_value\tcount\tcount1\tcount2\tcount3\tcount4\tmax\tmin\n" +
-                            "1970-01-01T00:00:00.189996Z\t22499\t89996\t49996.0\t3.999729996E9\t9996.0\t80001\t80001\t80001\t80001\t80001\t89996.0\t9996.0\n" +
-                            "1970-01-01T00:00:00.189997Z\t22499\t89997\t49997.0\t3.999809997E9\t9997.0\t80001\t80001\t80001\t80001\t80001\t89997.0\t9997.0\n" +
-                            "1970-01-01T00:00:00.189998Z\t22499\t89998\t49998.0\t3.999889998E9\t9998.0\t80001\t80001\t80001\t80001\t80001\t89998.0\t9998.0\n" +
-                            "1970-01-01T00:00:00.189999Z\t22499\t89999\t49999.0\t3.999969999E9\t9999.0\t80001\t80001\t80001\t80001\t80001\t89999.0\t9999.0\n" +
-                            "1970-01-01T00:00:00.190000Z\t22500\t90000\t50000.0\t4.00005E9\t10000.0\t80001\t80001\t80001\t80001\t80001\t90000.0\t10000.0\n",
+                    "ts\ti\tj\tavg\tsum\tfirst_value\tfirst_not_null_value\tcount\tcount1\tcount2\tcount3\tcount4\tmax\tmin\n" +
```

Comment:
the source dataset does not have null values in double column, we need a better test(s)